### PR TITLE
Some doc fixes to the new types/functions

### PR DESCRIPTION
**Pull request by tronical**

### CODE REVIEW

File: internal/core/api.rs [hunk 1]
```diff
@@ -383,23 +383,42 @@ impl Window {
...
+        /// The button that was pressed.
+        button: PointerEventButton,
+    },
+    /// A pointer was released.
+    PointerReleased {
+        position: LogicalPosition,
+        /// The button that was released.
+        button: PointerEventButton,
+    },
+    /// The position of the pointe has changed.
```
Comment (ogoffart) [inline internal/core/api.rs:411]:
```suggestion
    /// The position of the pointer has changed.
```

File: internal/core/api.rs [hunk 2]
```diff
@@ -383,23 +383,42 @@ impl Window {
...
+/// A event that describes user input.
+///
+/// Slint backends typically receive events from the windowing system, translate them to this
+/// enum and deliver to the scene of items via [`Window::dispatch_event()`].
+///
+/// The pointer variants describe events originating from an input device such as a mouse
+/// or a contact point on a touch-enabled surface.
+///
+/// All position fields are in logical window coordinates.
 #[allow(missing_docs)]
```
Comment (ogoffart) [inline internal/core/api.rs:395]: maybe not needed anymore?

**Review by ogoffart — APPROVED** [review]

File: internal/core/api.rs (hunk 1 above)
Comment (tronical) [reply internal/core/api.rs:411]: Thanks :)

File: internal/core/api.rs (hunk 2 above)
Comment (tronical) [reply internal/core/api.rs:395]: I just checked, it's still needed :( - unless we duplicate the position fields. I'll keep it for now.

**Pull request merged**